src/fontinfo.rs

```diff
@@ -1353,16 +1353,16 @@ impl<'de> Deserialize<'de> for Os2Panose {
 impl From<Os2PanoseV2> for Os2Panose {
     fn from(value: Os2PanoseV2) -> Self {
         Os2Panose {
-            family_type: value.family_type.abs() as NonNegativeInteger,
-            serif_style: value.serif_style.abs() as NonNegativeInteger,
-            weight: value.weight.abs() as NonNegativeInteger,
-            proportion: value.proportion.abs() as NonNegativeInteger,
-            contrast: value.contrast.abs() as NonNegativeInteger,
-            stroke_variation: value.stroke_variation.abs() as NonNegativeInteger,
-            arm_style: value.arm_style.abs() as NonNegativeInteger,
-            letterform: value.letterform.abs() as NonNegativeInteger,
-            midline: value.midline.abs() as NonNegativeInteger,
-            x_height: value.x_height.abs() as NonNegativeInteger,
+            family_type: value.family_type.unsigned_abs(),
+            serif_style: value.serif_style.unsigned_abs(),
+            weight: value.weight.unsigned_abs(),
+            proportion: value.proportion.unsigned_abs(),
+            contrast: value.contrast.unsigned_abs(),
+            stroke_variation: value.stroke_variation.unsigned_abs(),
+            arm_style: value.arm_style.unsigned_abs(),
+            letterform: value.letterform.unsigned_abs(),
+            midline: value.midline.unsigned_abs(),
+            x_height: value.x_height.unsigned_abs(),
```

Comment:
are these changes related to this patch?

Reply:
ahhh okay I get it, I was confused by `NonNegativeInteger`.

As a general thing I like to do clippy fixups as separate patches, since that way if we end up wanting to revert a commit we don't suddenly also get clippy fixes rolled-back, but no worries :)